JSDemos/Demos/List/ListEditingAndAPI/description.md

```diff
@@ -1 +1,23 @@
-The **List** is&nbsp;a&nbsp;UI component that represents a&nbsp;collection of&nbsp;items in&nbsp;a&nbsp;scrollable list. This demo displays different delete modes and shows how you can handle the **selectionChanged** event.
\ No newline at end of file
+The [List](/Documentation/ApiReference/UI_Components/dxList/) displays a collection of items as a scrollable list. To create a list, pass an array of items to the [dataSource](/Documentation/ApiReference/UI_Components/dxList/Configuration/#dataSource) property. 
+
+To allow users to delete items, set the [allowItemDeleting](/Documentation/ApiReference/UI_Components/dxList/Configuration/#allowItemDeleting) property to **true**. Use the "Allow deletion" checkbox under the List to see how this property works.
```

Comment:
"to see how this property works" instead of saying what you should see, it is better to state what something does.

Reply:
Could you provide an example, please?

Reply:
The "Allow deletion" checkbox under the List does this or that... (what does it do? Saying that it lets you see how something works does not explain what this checkbox does) Does it change something?